doc/src/user_guide.rst

```diff
@@ -101,8 +101,8 @@ When installing from source, to manage PyFR's Python dependencies we
 strongly recommend using `pip <https://pypi.python.org/pypi/pip>`_ and
```

Comment:
When installing from source, we strongly recommend using ... to manage the Python dependencies.